.circleci/conditional_config.yml

```diff
@@ -173,7 +181,7 @@ jobs:
     
   checkout_code:
     docker:
-      - <<: *frontend_base
+      - <<: *frontend_ci
```

Comment:
we don't need chromium here

Reply:
I must say I find the naming a bit confusing. We use `frontend_base` also in the CI. As many others of the images. CI isn't what makes this image different.

Also why is the `frontend_ci` image not based on the `frontend_base` image?

Reply:
Ah sorry, the other way around.

Reply:
now, `frontend_base` is based on `frontend_ci`